src/app/book/list/list.component.html

```diff
@@ -2,28 +2,43 @@
 
   <md-table #table [dataSource]="dataSource">
 
-    <!--- Note that these columns can be defined in any order.
-          The actual rendered columns are set as a property on the row definition" -->
-
     <!-- ID Column -->
     <ng-container cdkColumnDef="id">
-      <md-header-cell *cdkHeaderCellDef> ID </md-header-cell>
-      <md-cell *cdkCellDef="let row"> {{row.id}} </md-cell>
+      <md-header-cell *cdkHeaderCellDef [ngClass]="'smallWidth'"> ID </md-header-cell>
```

Comment:
The [ngClass] directive is only useful in expressions.  You can easily use the standard HTML class attribute.